src/Core/SettingsChangesHistory.cpp

```diff
@@ -57,6 +57,7 @@ String ClickHouseVersion::toString() const
 /// Note: please check if the key already exists to prevent duplicate entries.
 static std::initializer_list<std::pair<ClickHouseVersion, SettingsChangesHistory::SettingsChanges>> settings_changes_history_initializer =
 {
+    {"24.8", {{"query_plan_convert_outer_join_to_inner_join", true, false, "A bug was found."}}},
```

Comment:
Let's do `false`, `false` here and in the changes for 24.4 so we can backport it and disable it by default everywhere.